app/views/projects.html

```diff
@@ -8,52 +8,201 @@
+                <div class="pad-top-lg pad-left-xl pad-right-xl">
+                  <span class="h1">Projects</span>
+                  <a ng-if="canCreate" href="create-project" class="visible-xs btn btn-md btn-primary pull-right">
+                    <span>New project</span>
+                  </a>
+                </div>
 
-          <div ng-if="!showGetStarted" class="container surface-shaded"> <!-- Safari needs surface-shaded at mobile -->
-            <div ng-if="(projects | hashSize) === 0" class="text-muted" style="margin-top: 50px;">Loading...</div>
-            <div ng-if="(projects | hashSize) !== 0" class="gutter-top">
-              <h1 style="display: inline-block;">Projects</h1>
-              <a ng-if="canCreate" href="create-project" style="margin-top: 10px;" class="btn btn-lg btn-primary pull-right">New Project</a>
-              <alerts alerts="alerts"></alerts>
-              <div ng-repeat="project in projects | orderByDisplayName track by project.metadata.name"
-                   class="animate-repeat">
-                <div row flex cross-axis="center" class="tile tile-project tile-click tile-flex">
-                  <div flex class="project-summary">
-                    <h2 class="project truncate">
-                      <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | uniqueDisplayName : projects)}}</a>
-                      <span ng-if="project.status.phase != 'Active'" data-toggle="tooltip" title="This project has been marked for deletion." class="pficon pficon-warning-triangle-o" style="cursor: help; vertical-align: top; margin-left: 5px;"></span>
-                    </h2>
+                <div row cross-axis="center" class="project-bar pad-lg table-toolbar form-inline">
+                  <input class="projectSearch" placeholder="Search projects" ng-model="filterStr">
+
+                  <span class="mar-left-sm mar-right-sm">
+                    <span class="vertical-divider"></span>
+                  </span>
+
+                  <div class="project-sort">
+                    <div pf-sort config="sortConfig" class="sort-controls"></div>
+                  </div>
+
+                  <span class="hidden-xs mar-left-sm mar-right-sm">
+                    <span class="vertical-divider"></span>
+                  </span>
+
+                  <span flex class="hidden-xs"></span>
+
+                  <div>
+                    <a ng-if="canCreate" href="create-project" class="hidden-xs btn btn-md btn-primary pull-right">
+                      <span>New project</span>
+                    </a>
+                  </div>
+                  <alerts alerts="alerts"></alerts>
+                </div>
+              </div>
+
+              <div ng-repeat="project in projects | findProject:filterStr">
+                <div row cross-axis="center"
+                     class="tile tile-project tile-click tile-flex tile-project-border mar-left-xl mar-right-xl mar-bottom-none">
```

Comment:
Prob same as above, I like the generic classes, but prob after ~3 classes need to consider making a specific class to roll some of that up (and hopefully then make it reusable).